test/appium/tests/atomic/chats/test_public.py

```diff
@@ -328,6 +329,304 @@ def test_public_chat_open_using_deep_link(self):
+                self.errors.append("Counter of reaction is not set on %s for message receiver!" % key)
+            self.chat_1.set_reaction(key)
+            if message.emojis_below_message(own=False) == 1:
+                self.errors.append("Counter of reaction is not re-set on %s for message receiver!" % key)
+
+        self.chat_2.just_fyi("Sending Emoji/Tag/Links in chat")
+        # TODO: add link and tag messages after #11168 is fixed(rechecked 23.11.21, valid)
+        navigate_to_start_state_of_both_devices()
+
+        emoji_name = random.choice(list(emoji.EMOJI_UNICODE))
+        emoji_unicode = emoji.EMOJI_UNICODE[emoji_name]
+
+        self.chat_2.just_fyi("Sending Emoji in chat")
+        self.chat_2.chat_message_input.send_keys(emoji.emojize(emoji_name))
+        self.chat_2.send_message_button.click()
+
+        if not self.device_1.element_by_text_part(emoji_unicode).is_element_displayed(10):
+            self.errors.append("Emoji not appeared in Push Notification")
+            self.device_1.click_system_back_button()
+            self.device_1.get_app_from_background()
+        else:
+            self.device_1.element_by_text_part(emoji_unicode).click()
+
+        emoji_message = self.chat_2.chat_element_by_text(emoji_unicode)
+        self.chat_1.set_reaction(emoji_unicode, emoji_message=True)
+        if emoji_message.emojis_below_message(own=False) != 1:
+            self.errors.append("Counter of reaction is not set on Emoji for message receiver!")
+        self.chat_1.set_reaction(emoji_unicode, emoji_message=True)
+        if emoji_message.emojis_below_message(own=False) == 1:
+            self.errors.append("Counter of reaction is not re-set on Emoji for message receiver!")
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702067)
+    @marks.medium
+    def test_messaging_open_user_profile_long_press_on_message_1_1_chat(self):
+
+        message_from_sender = "Message sender"
+
+        self.home_1.just_fyi("1-1 chat: sender adds receiver and send a message")
+        self.chat_1.send_message(message_from_sender)
+        self.chat_1.chat_element_by_text(message_from_sender).long_press_element()
+        if self.chat_1.view_profile_by_avatar_button.is_element_displayed():
+            self.errors.append('Member photo is shown on long tap on sent message from 1-1 chat')
+        self.chat_1.click_system_back_button(2)
+
+        self.home_2.just_fyi("1-1 chat: receiver verifies that can open sender profile on long tap on message")
+        self.chat_2.chat_element_by_text(message_from_sender).long_press_element()
+        if self.chat_2.view_profile_by_avatar_button.is_element_displayed():
+            self.errors.append('1-1 chat: another user profile is opened on long tap on received message')
+        self.chat_2.click_system_back_button(2)
+
+    @marks.testrail_id(702068)
+    @marks.medium
+    def test_messaging_open_user_profile_long_press_on_message_in_public_chat(self):
```

Comment:
not sure it is worth separate case;
can be added as a check to any public chat test and removed.

Reply:
to say in truth, I would prefer to leave it as separate case than moving it as check to some other irrelevant public test. IMHO, adding this test as additional check in other test case goes against our goal of making more simple and atomic cases.  But if you insist, I can move this check to @marks.testrail_id(700734) test_public_message_edit. What do you think?

Reply:
decided to remove this check at all